tests/params_test.py

```diff
@@ -145,12 +145,14 @@ def param_test_generator(tmp_path, param, value):
     ui[param]["enabled"] = True
```

Comment:
If you add a ui["geoh5"] = None here _init_params will bypass another time-consuming Workspace creation.